boa3_test/tests/boatestcase.py

```diff
@@ -294,6 +298,27 @@ async def deploy(
 
             return cls.deployed_contracts[_manifest.name]
 
+    @classmethod
+    async def get_storage(
+            cls,
+            prefix: Optional[bytes] = None,
+            *,
+            target_contract: Optional[types.UInt160] = None,
+            remove_prefix: bool = False,
+            key_post_processor: Optional[PostProcessor] = None,
+            values_post_processor: Optional[PostProcessor] = None,
+    ) -> dict[bytes, bytes]:
+        try:
+            return await super().get_storage(
+                prefix,
+                target_contract=target_contract,
+                remove_prefix=remove_prefix,
+                key_post_processor=key_post_processor,
+                values_post_processor=values_post_processor
+            )
+        except TypeError:
+            return {}
```

Comment:
Why return an empty dict when there is a TypeError? Isn't it better to let it throw an Error and let the user catch it later?

Reply:
Because the error happens when the storage is empty, I don't think it should throw an error when an empty storage is an expected output.
The error is on boa test constructor, so we can only handle the error on our side

Reply:
Just to make sure, if I use the wrong PostProcessor, the error won't be a TypeError?
Like using `values_post_processor=storage.as_public_key` when the value on the storage is not a PublicKey